app/src/main/java/zapsolutions/zap/setup/ConnectRemoteNodeActivity.java

```diff
@@ -192,23 +195,29 @@ private void connect(RemoteConfiguration config) {
             if (config instanceof LndConnectConfig) {
                 LndConnectConfig lndConfig = (LndConnectConfig) config;
 
-                walletConfigsManager.addWalletConfig(WalletConfigsManager.DEFAULT_WALLET_NAME,
-                        "remote", lndConfig.getHost(), lndConfig.getPort(),
+                String id;
+                id = walletConfigsManager.addWalletConfig(config.getHost(),
```

Comment:
Any reason you chose to separate those lines and not just do:
`String id = walletConfigsManager.add`